.github/workflows/ci.yml

```diff
@@ -39,20 +39,31 @@ jobs:
       - name: Run ShellCheck
         uses: ludeeus/action-shellcheck@master
         with:
-            ignore_paths: ./.devcontainer
+          ignore_paths: ./.devcontainer
 
   test-install:
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        shell: [sh, bash, zsh]
+        shell: [sh, bash, zsh, fish]
```

Comment:
You should not do these (just) by "shell" you should (also) do them also by "approach". In many cases the shell does not matter.

Reply:
updated